inst/tests/tests.Rraw

```diff
@@ -14900,6 +14909,23 @@ test(2050.4, rbind(DT[0], DT[0])[,levels(f)], c("a","b","c"))  # now ok again (o
 test(2050.5, rbindlist(list(DT[0], DT[0]))[,levels(f)], c("a","b","c"))  # now ok again
 test(2050.6, rbind(DT[1], data.table(f=factor(letters[10:11]))[0])[,levels(f)], c("a","b","c","j","k")) # now includes "j","k" again
 
+# coverage tests
+## abusing -.Date, format.data.table, rleidv
+test(2051.1, `-.IDate`(.Date(0), 1L), .Date(-1))
+test(2051.2, `-.IDate`(1L, 1L), error = 'only subtract from "IDate"')
+
+test(2051.3, format.data.table(1L), error = 'Possibly corrupt data.table')
+
+test(2051.4, rleidv(prefix = 1L), error = 'prefix must be NULL or')
+
+## passing Date to second argument of as.POSIXct.ITime
+t = as.ITime(0L)
+test(2051.5, as.POSIXct(t, .Date(0L)), .POSIXct(0, 'UTC'))
+
```

Comment:
Please can you remove these 4 blank lines.  The style in the file is that a group of tests like 2051.1 - 2051.6 should appear together as a block.

Reply:
Usually yes, in this case, it's coverage tests & the file being covered is different. Doesn't feel like a new test number is needed but some visual space made sense. can revert though

Reply:
It's a nice single block of coverage tests to my eyes.  Which is in an unrelated PR -- which is more to the point.  In a wide-ranging very high line number PR like this,  I was hoping to see there were no changes to tests at all.  But then I saw there were changes to tests.  But then I saw it was just a bit of extra coverage in a single block,  so that's ok.   That's the way I looked at it.

Reply:
Yea, makes sense. The coverage stuff is because Codecov complained on first commit since the changes hit some uncovered lines so the patch diff failed. So I took a pass at some lines that stuck out in a scan of the coverage diff.

I made sure to file the coverage stuff in a different commit, so in git history at least it's easier to see. But if anyone looks at this PR as a source, harder to figure out what's going on.

Reply:
Oh I see now, thanks.  Makes perfect sense.